apps/src/metrics/AnalyticsConstants.js

```diff
@@ -187,6 +187,7 @@ const EVENTS = {
   // Levels
   FEEDBACK_SUBMITTED: 'Level Feedback Submitted',
   RUBRIC_LEVEL_VIEWED_EVENT: 'Rubric Level Viewed',
```

Comment:
Down the road, we should rename this variable to `RUBRIC_LEVEL_VIEWED` for consistency.

Reply:
This would be out of the scope of this PR, but we can come back and rename these if we'd like in a followup PR.